tests/errors/dataclass_e5.pyx

```diff
@@ -15,7 +15,7 @@ cdef class C:
 _WARNINGS = """
 9:7: Found Python 2.x type 'long' in a Python annotation. Did you mean to use 'cython.long'?
 10:7: Found C type 'Py_ssize_t' in a Python annotation. Did you mean to use a Python type?
```

Comment:
I wonder if this suggestion even makes sense. Users probably meant to use a `cython.*` type here, not a Python type.

Reply:
> I wonder if this suggestion even makes sense.
If I understand this comment is connected to this line:
```
10:7: Found C type 'Py_ssize_t' in a Python annotation. Did you mean to use a Python type?
```
Should I fix it in this PR?

Reply:
I think it'd be in scope to improve it in the PR.

Reply:
Fixed to : `Found C type 'Py_ssize_t' in a Python annotation. Did you mean to use a 'cython.Py_ssize_t'?` But @da-woods does this warning have any meaning except unitests? `warning()` call is missing `level=` hence it is raised **only during unittests**:

```
(cython39) matus@MacBook-Pro:~/dev/cython/test$ cat test.py
import cython

def main():
    a: Py_ssize_t = 6
(cython39) matus@MacBook-Pro:~/dev/cython/test$ cython -3 test.py
warning: test.py:4:7: Unknown type declaration 'Py_ssize_t' in annotation, ignoring
```

Reply:
Sorry for the slow reply...

Without the `level` call warnings aren't shown by default. But they aren't just for the CI. You can turn them on with compiler options. I think `show_all_warnings` as [a Cythonize argument](https://cython.readthedocs.io/en/latest/src/userguide/source_files_and_compilation.html#cythonize-arguments) will do it.

So it isn't *just* for the unittests, but most people probably don't see them. As I said, it'd be welcome, but not necessary for this PR.

Reply:
> So it isn't _just_ for the unittests, but most people probably don't see them. As I said, it'd be welcome, but not necessary for this PR.

OK fair enough. I learn new think every day. The message is changed in PR so I am closing this conversation.